### PR TITLE
 fragments/rgw: added rgw fragments

### PR DESCRIPTION
**Pull request by udaysk23**
Added RGW fragments and created T0 directory in suite/nautilus/rgw/t0/

### CODE REVIEW

**Comment by udaysk23** [comment]
Will add TC logs shortly

File: fragments/rgw/test_versioning_objects_enable.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+tests:
```
Comment (psathyan) [inline fragments/rgw/test_versioning_objects_enable.yaml:1]: We could further structure the folder to `rgw/versioning/test_objects_enable.yaml`

**Comment by psathyan** [comment]
Sorry, the guidelines are not available yet. Hence, we can discuss on how we want to do this

- All fragments must live in fragments folder at root level
- Group fragments into containers/folders/dirs based on purpose, feature, etc
- Descriptive fragment name
- At all costs, avoid duplicates
- Ensure each fragment has a Polarion ID

**Comment by rakeshgm** [comment]
> Sorry, the guidelines are not available yet. Hence, we can discuss on how we want to do this
> 
> * All fragments must live in fragments folder at root level
> * Group fragments into containers/folders/dirs based on purpose, feature, etc
> * Descriptive fragment name
> * At all costs, avoid duplicates
> * Ensure each fragment has a Polarion ID

I think we should rephrase the meaning of the last point. so there may be a fragment which is not a test case, for example, install pre-req and cluster-health check frags. these don't need polarion id. I think we should make polarion id's as mandatory if the fragment is a test. 

I agree with all the other points

**Comment by udaysk23** [comment]
Due to commit issue, Closing this PR

**Pull request closed without merging**